src/styles/styles.js

```diff
@@ -1965,6 +1969,7 @@ const styles = {
         fontSize: variables.iouAmountTextSize,
         color: themeColors.heading,
         padding: 0,
+        lineHeight: undefined,
```

Comment:
Why do we add this?

Reply:
To reset the line height. TextInput on the IOU page was not using any lineHeight. But BaseTextInput has it set to 20. Font size for IOU amount input is bigger than the normal input thus this line-height is wrong.

Reply:
Hmm, I am only applying this to IOU input via `styles.iouAmountTextInput`. So it should not affect any other part of the app.

>  It still feels a bit strange to me to tie that to borderRight 

I didn't get this.